Lib/LinkedIn/LinkedInApi.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+
+use GuzzleHttp\Client;
+
+class LinkedInApi {
```

Comment:
- Je eerste result wordt bijna niet onderhouden
- Er is ook https://github.com/Happyr/LinkedIn-API-client, maar dit ziet er echt onnodig complex uit en uiteindelijk moet je alsnog zelf `->post('v1/people/~/shares', $options)` doen.